client/packages/programs/src/JsonForms/common/components/Text.tsx

```diff
@@ -17,6 +17,8 @@ const Options = z
      * Additional pattern to be matched that can be defined in ui schema
      */
     pattern: z.string().optional(),
+    /** Overwrite the schema definition, e.g. to make a non-required field required in the UI. */
```

Comment:
does this mean: 'Add a required property to allow text fields to be mandatory in the UI'?
It took me some time to understand what the comment was saying.

Reply:
that's a good point - having optional data be required in the UI is inconsistent. Better to just change the requirement in DefaultPatientSchema